backend/app/modules/multisig_signatures/service/offchain.py

```diff
@@ -16,8 +20,11 @@
 
 
 class OffchainMultisigSignatures(Model):
+    is_mainnet: bool = False
     verifiers: dict[SignatureOpType, Verifier]
 
+    staged_signatures: list[MultisigSignatures] = []
```

Comment:
I think this might be error prone (race condition) to keep a state shared between requests here in services and I'd like to avoid it. However, it's not blocking because atm we have a limited number of users with multisigs

Reply:
Could you create a task and leave a TODO here to change it?